elasticsearch-persistence/test/unit/repository_response_results_test.rb

```diff
@@ -45,6 +45,14 @@ class MyDocument; end
       assert_equal 5, subject.response._shards.total
     end
 
+    should "provide access to the raw respanse" do
```

Comment:
There is a typo with `respanse` instead of `response` right ?

Reply:
@guillaumebriday Thanks for noticing. Fixed.